packages/react-chat/src/components/Image/Default.tsx

```diff
@@ -17,6 +17,7 @@ export const DefaultImageBase = styled(tag('img'), {
         width: 248,
       },
       embedded: {
+        width: '100%',
```

Comment:
images need to stretch to fit the width, but only up to 512px.
Without this, even when you shrink the window down (under 512 px), the image will go off the screen.